src/firefly/js/visualize/saga/MouseReadoutWatch.js

```diff
@@ -400,7 +398,7 @@ function makeWLResult(plot,imagePt= undefined) {
         }
     }
     else {
-        return;
+        return undefined;
```

Comment:
Is there any difference is the 2 methods?

Reply:
i like `return undefined` because it is better documented.